### PR TITLE
feat(mep): Introduce the count_web_vitals functions to metrics

### PR DESCRIPTION
**Pull request by wmak**
- These are the equivalent metrics function versions of what was added
  in discover

### CODE REVIEW

File: src/sentry/search/events/datasets/metrics.py [hunk 1]
```diff
@@ -276,3 +287,44 @@ def _resolve_percentile(
...
+        ]:
+            raise InvalidSearchQuery("count_web_vitals only supports measurements")
+
+        measurement_rating = Column(f"tags[{indexer.resolve('measurement_rating')}]")
+
+        quality_id = indexer.resolve(quality)
+        if quality_id is None:
+            # Choose an invalid id so the result is 0
+            # TODO: Should be able to just not return a function at all in these cases
+            quality_id = 0
```
Comment (Zylphrex) [inline src/sentry/search/events/datasets/metrics.py:315]: Can you just return a stub that always resolves to 0 to save computation? like `toUint64(0)`?

File: src/sentry/search/events/datasets/metrics.py [hunk 2]
```diff
@@ -276,3 +287,44 @@ def _resolve_percentile(
...
+        if column not in [
+            "measurements.lcp",
+            "measurements.fcp",
+            "measurements.fp",
+            "measurements.fid",
+            "measurements.cls",
+        ]:
+            raise InvalidSearchQuery("count_web_vitals only supports measurements")
+
+        measurement_rating = Column(f"tags[{indexer.resolve('measurement_rating')}]")
```
Comment (Zylphrex) [inline src/sentry/search/events/datasets/metrics.py:309]: Should this be some kind of helper function?

**Review by k-fish — APPROVED** [review]
Other comments are 👍  but functionality seems fine

File: src/sentry/search/events/datasets/metrics.py (hunk 2 above)
Comment (wmak) [reply src/sentry/search/events/datasets/metrics.py]: :+1: yeah dunno why i forgot about using the builders column resolver here

File: src/sentry/search/events/datasets/metrics.py (hunk 1 above)
Comment (wmak) [reply src/sentry/search/events/datasets/metrics.py]: Good call, I was thinking to just not even create a function in these cases but that's a good solution that's far easier to implement

**Pull request merged**